### PR TITLE
chore: add CDK and zip lambda examples

### PR DESCRIPTION
**Pull request by cprice404**
This commit adds two new example projects; both illustrate how
to deploy an AWS lambda function that uses the Momento Rust SDK.
The first example uses CDK to deploy the function; the second
simply uses `cargo-lamba` to build a zip file, and then gives
the user instructions on how to upload it via the AWS console.

### CODE REVIEW

File: example/aws/cdk-lambda/README.template.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- To get started with Momento you will need a Momento API key. You can get one from the [Momento Console](https://console.gomomento.com).
+
+## Building and Deploying the Lambda Function
+
+To build and deploy the Lambda function, first make sure that your AWS credentials are set up properly (via env vars or
+~/.aws/credentials file). Then all you need to do is run the following commands:
+
+```bash
+cd infrastructure
+npm ci -- deploy --parameters MomentoApiKey=<YOUR_MOMENTO_API_KEY>
```
Comment (anitarua) [inline example/aws/cdk-lambda/README.template.md:23]: should be `npm ci && npm run cdk` here I think?

Comment (cprice404) [reply example/aws/cdk-lambda/README.template.md:23]: lol thanks

Comment (cprice404) [inline example/aws/cdk-lambda/README.template.md:23]:
```suggestion
npm ci
npm run cdk -- deploy --parameters MomentoApiKey=<YOUR_MOMENTO_API_KEY>
```

**Review by anitarua — DISMISSED** [review]
v cool, they're so fast!

**Review by anitarua — APPROVED** [review]

**Pull request merged**